terraform/main.tf

```diff
@@ -177,6 +178,15 @@ variable "pubsub_functions" {
       trigger_event = "update_user_role"
       secrets       = ["DISCORD_TOKEN"]
     },
+    welcome : {
```

Comment:
As discussed on Discord, we won't need a new function, we can use the `channelMessage` function

Reply:
Yeah, I will remove Welcome function since it will be useless and bring the change in terraform/main.tf with it.